contracts/SimpleOperations.sol

```diff
@@ -45,121 +46,258 @@ contract SimpleOperations is Operations {
 	uint32 public latestFork = 0;
 	address public grandOwner = msg.sender;
 
-	function SimpleOperations() public {
+	function SimpleOperations()
+		public
+	{
 		client["parity"] = Client(msg.sender);
 		clientOwner[msg.sender] = "parity";
 	}
 
-	function() public payable {
-		Received(msg.sender, msg.value, msg.data);
-	}
-
 	// Functions for client owners
 
-	function setClientOwner(address _newOwner) public only_client_owner {
-		var newClient = clientOwner[msg.sender];
-		clientOwner[msg.sender] = bytes32(0);
+	function setClientOwner(address _newOwner)
```

Comment:
Setting a client owner to `0x0` cause the client to be treated as removed (see comment in `clientExists()`). Best to disallow setting it to `0x0`